### PR TITLE
Get the JWT for unit tests from FlightPassport

### PR DESCRIPTION
**Pull request by cyapa**
## Proposed changes

[JIRA](https://ssrc.atlassian.net/browse/SCUTM-354)

Removed the hardcoded JWT from API unit tests. Now the tests are retrieving the JWT from FlightPassport service.
Used environment for FlightPassport :`https://passport.utm.dev.airoplatform.com`

README update PR in USSP_LINK: https://github.com/tiiuae/drone-utm/pull/34

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply._

- [x] Lint and unit tests pass locally with the changes
- [ ] Added new tests that prove the changes in this PR works
- [x] Added necessary documentation (if appropriate)
- [ ] Added copyrights to new files (if appropriate)

## Further comments

❗❗ Already updated the `ENV_FILE` secret we have in this repo with the new env changes

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
+        "scope": "blender.write blender.read",
+        "audience": env.get("PASSPORT_AUDIENCE", ""),
+    }
+    token_url = env.get("PASSPORT_URL", "") + env.get("PASSPORT_TOKEN_URL", "")
+    response = requests.post(token_url, data=data)
+
+    # Check for a successful response and return the token
+    if response.status_code == status.HTTP_200_OK:
+        token_data = response.json()
+        return token_data.get("access_token")
```
Comment (govindsi) [inline conftest.py:27]: Validate the token using pyJWT , May be it go in other PR also

File: flight_feed_operations/test_views.py [hunk 2]
```diff
@@ -9,7 +9,9 @@
 from rest_framework import status
 from rest_framework.test import APITestCase
 
-JWT = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJ0ZXN0ZmxpZ2h0LmZsaWdodGJsZW5kZXIuY29tIiwiY2xpZW50X2lkIjoidXNzX25vYXV0aCIsImV4cCI6MTY4Nzc4Mjk0OCwiaXNzIjoiTm9BdXRoIiwianRpIjoiODI0OWI5ODgtZjlkZi00YmNhLWI2YTctODVhZGFiZjFhMTUwIiwibmJmIjoxNjg3Nzc5MzQ3LCJzY29wZSI6ImJsZW5kZXIucmVhZCIsInN1YiI6InVzc19ub2F1dGgifQ.b63qZWs08Cp1cgfRCtbQfLom6QQyFpqUaFDNZ9ZdAjSM690StACij6FiriSFhOfFiRBv9rE0DePJzElUSwv1r1bI0IpKMtEJYsJY4DXy7ZImiJ3rSten1nnb1LLAELcDIxMZM2D1ek43EFW35al4si640JfMcSmt62bEP1b4Msc"
+from conftest import get_oauth2_token
+
+JWT = get_oauth2_token()
```
Comment (govindsi) [inline flight_feed_operations/test_views.py:14]: can this be moved to setup?

**Review by govindsi — APPROVED** [review]
LGTM

File: flight_feed_operations/test_views.py (hunk 2 above)
Comment (cyapa) [reply flight_feed_operations/test_views.py:14]: Since `setUp()` runs for each test case we cannot move it inside :)

**Pull request merged**